test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -0,0 +1,129 @@
+﻿function Write-LogPassThru
+{
+    Param(
+        [Parameter(Mandatory=$true, ValueFromPipeline=$true, Position = 0)]
+        [string] $Message,
+        $Path = "$env:Temp\CodeCoverageRunLogs.txt"
+    )
+
+    $message = "$([Datetime]::Now.ToShortDateString()) - $([datetime]::Now.ToShortTimeString()) : $message"
```

Comment:
>$message = "$([Datetime]::Now.ToShortDateString()) - $([datetime]::Now.ToShortTimeString()) : $message" [](start = 4, length = 103)

this can also be done as:
`"{0:d} - {0:t} : {1}" -f ([datetime]::now),$message`
or even:
`"{0} : {1}" -f ([datetime]::now),$message`